### PR TITLE
Rename 'Salesforce DX CLI' >> 'Salesforce CLI'

### PR DESCRIPTION
**Pull request by guw**
### What does this PR do?
$subject

### What issues does this PR fix or reference?
@W-5004613@

### CODE REVIEW

**Comment by guw** [comment]
@vazexqi FYI

File: packages/salesforcedx-vscode-core/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ To run a command from Salesforce Extensions for VS Code, press Cmd+Shift+P (macO
 When a command finishes running (due to success, failure, or cancellation), a notification displays at the top of the window.  
 ![Notification that source was successfully pushed to a scratch org](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-core/images/command_success_notification.png)
 
-To see the output of the commands that you run, select **View** > **Output**, and then select **Salesforce DX CLI** from the dropdown menu. Alternatively, click **Show** in the completion notification.  
+To see the output of the commands that you run, select **View** > **Output**, and then select **Salesforce CLI** from the dropdown menu. Alternatively, click **Show** in the completion notification.
```
Comment (vazexqi) [inline packages/salesforcedx-vscode-core/README.md]: @ruthemmanuelle - Just FYI. Any comments here?

Comment (ruthemmanuelle) [reply packages/salesforcedx-vscode-core/README.md]: Thanks for fixing this, Gunnar.

**Comment by guw** [comment]
@vazexqi I updated the branch. Please feel free to merge it at your convenience once the build is passing.

**Pull request merged**